docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,194 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
+
+Node affinity, described earlier, allows a pod to "be attracted to" a set of nodes
+(either as a preference or a hard requirement). Taints are the opposite -- they allow
+a *node* to *repel* a set of pods.
+
+More conceretely: *taints* go on nodes, and *tolerations* go on pods. A taint on a node
+means "repel all pods that do not toleate the taint."
+
+You add a taint to a node using `kubectl taint`. For example,
+
+```shell
+kubectl taint nodes node1 key=value:NoSchedule
+```
+
+places a taint on node `node1`. The taint has key `key`, value `value`, and taint effect `NoSchedule`.
+This means that no pod will be able to schedule on `node1` unless it has a matching toleration.
+The toleration is specified in the pod template. Both of the following tolerations "match" the
+taint created by the `kubectl taint` line above, and thus a pod with either toleration would be able
+to schedule onto `node1`:
+
+```yaml
+tolerations: 
+   - key: "key"
+     operator: "Equal"
+     value: "value"
+     effect: "NoSchedule"
+```
+
+```yaml
+tolerations: 
+   - key: "key"
+     operator: "Exists"
+     effect: "NoSchedule"
+```
+
+A toleration "matches" a taint if the `key`s are the same and the `effect`s are the same, and
+
+* the `operator` is `Exists` (in which case no `value` should be specified), or
+* the `operator` is `Equal` and the `value`s are equal
+
+(`Operator` defaults to `Equal` if not specified.)
+As a special case, an empty `key` with operator `Exists` matches all keys and all values.
+Also as a special case, empty `effect` matches all effects.
+
+The above example used `effect` of `NoSchedule`. Alternatively you can use `effect` of `PreferNoSchedule`
+which is a "preference" or "soft" version of `NoSchedule` -- the system will *try* to avoid placing a
```

Comment:
Thanks @davidopp , my thinking for when using backtick is that this may not limited to some `Kubenretes terminology`, but also some `key words` which we want to highlight something in the document, comments?